plugins/CoreHome/javascripts/refresh-button.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * @link http://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+$(function () {
+  var iconRefresh = $('.top_controls .icon-reload');
+
+  function refresh() {
+    var Matomo = window.CoreHome.Matomo;
+    var hashParsed = window.CoreHome.MatomoUrl.hashParsed.value;
+
+    Matomo.postEvent('loadPage', hashParsed.category, hashParsed.subcategory);
+  }
+
+  function isCoreHomeModuleActive() {
+    var search = window.CoreHome.MatomoUrl.parse(window.location.search.slice(1));
+    return search.module === 'CoreHome';
+  }
+
+  if (isCoreHomeModuleActive()) {
+    iconRefresh.show();
```

Comment:
Just a small note on that. Using `display:none` as default, but calling `show()` here, causes the icon to be visible also in mobile view. Which shouldn't be intended. The reason is, that `show()` will add a `display:inline`, which overwrites the `display:none` that is set for smaller resolutions. That looks a bit ugly.
A better possibility is to always add the class `hidden` and remove it if needed.  See https://github.com/matomo-org/matomo/blob/4.x-dev/tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png
@peterhashair you should have actually seen that before updating the screenshots.
I will create a PR to fix that.

Reply:
Just saw there was already a fix for it in https://github.com/matomo-org/matomo/pull/19289, so nvm.